CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ All notable changes to this project will be documented in this file.
 ### Documents
 - Fix in user docs forget to use options while using BatchWrite [#2169](https://github.com/pingcap/tispark/pull/2169)
 
+### Known issue
+- Spark-Sql's authentication and authorization through TiDB will affect other datasource such as hive.
```

Comment:
Please write an issue and link to it.  
Describe how this feature affect other datasources

Reply:
done